themes/custom/droopler_theme/scss/components/_d_p_side_by_side.scss

```diff
@@ -101,6 +148,13 @@ $d-p-side-by-side-image-background-padding-y: 130px !default;
       &::before {
         content: none;
       }
+      @each $name, $themeSettings in $d-p-side-by-side-image-single_text_block-themes {
+        &.theme-#{$name} {
+          &::before {
+            content: "";
+          }
+        }
+      }
```

Comment:
w sumie to nie kumam, więc zapytam, zanim w ciemno klepnę "approve"...

po co tu przepisywać ten `::before { content: none } ` dla wszystkich themów ? nie wystarczy to powyżej ?

Reply:
Ten `none` w 149 jest niestety mocniejszy więc muszę go tu jeszcze raz nadpisać bo inaczej jak będzie obraz to nam wyleci temat.